docs/_layouts/default.html

```diff
@@ -187,7 +187,11 @@
             {% include linter-rule-summary.html %}
           {% endif -%}
           <section id="linter-toc" class="docs-component-sidebar-toc">
-            {% include toc.html html=content h_min=2 h_max=3 baseurl=page.url %}
+              {% if page.index %}
+              {% include toc.html html=content h_min=2 h_max=3 %}
```

Comment:
Why do we _not_ want `baseurl` in this case? (Also, I am not sure off the top of my head what `page.index` means.)

Reply:
I am not sure either, but it fixed the links. Otherwise, for example, `installation` was linked to `https://googleapis.github.io/#installation`. `page.index` is similar `page.rule`, that I added a metadata in the `index.md`:
```
---
index:
---
```